pkg/cli/vendor_update.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kubebuilder/internal/config"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold"
+)
+
+func (c cli) newVendorUpdateCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "update",
+		Short: "Update vendor dependencies",
+		Long:  `Update vendor dependencies`,
+		Example: fmt.Sprintf(`Update the vendor dependencies:
+%s update vendor`, c.commandName),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cfg, err := config.Read()
+			if err != nil {
+				log.Fatal(err)
+			}
+			if err = scaffold.NewUpdateScaffolder(cfg).Scaffold(); err != nil {
+				log.Fatal(err)
+			}
+			return nil
+		},
+	}
+}
```

Comment:
Why did you remove all the homogeneization recently done to this command.
- Errors were wrapped with `updateError`
- If `cfg, err := config.Read()` was returning a file-doesn't-exist error (`os.IsNotExist(err)`), a better error message was being yielded.
- You are not validating that it may only be run in `v1`.

I know that some of the steps of the generic plugins are no-ops for this case, but I still think that we should keep all of them homogenized.

Reply:
Changes were lost in the rebase. Will update.